packages/reporter/src/commands/command.tsx

```diff
@@ -47,25 +48,29 @@ const shouldShowCount = (aliasesWithDuplicates: Array<Alias> | null, aliasName:
   return _.includes(aliasesWithDuplicates, aliasName)
 }
 
-const CommandColumn = observer(({ model }) => {
-  if (model.hasChildren) {
-    return (
-      <div className='command-expander-column'>
-        <ChevronIcon
-          className={cs('command-expander', { 'command-expander-is-open': model.hasChildren && !!model.isOpen })}
-        />
-      </div>
-    )
-  }
-
-  return (
-    <div className='command-number-column'>
-      {<PinIcon className='command-pin' />}
+/**
+ * NavColumns Rules:
+ *   > Command Number Column
+ *      - When the command is executing, it is pending and renders the pending icon
+ *      - When the command is finished, it displays the command number
+ *        - Commands will render a command number, where Events and System logs do not
+ *      - When the command is finished and the user has pinned the log, it displays the pin icon
+ *
+ *   > Expander Column
+ *      - When the log is a group log and it has children logs, it will display the chevron icon
+ */
+const NavColumns = observer(({ model, isPinned, toggleColumnPin }) => (
+  <>
+    <div className='command-number-column' onClick={toggleColumnPin}>
       {model._isPending() && <RunningIcon className='fa-spin' />}
-      {!model._isPending() && <span className='command-number'>{model.number || ''}</span>}
+      {(!model._isPending() && isPinned) && <PinIcon className='command-pin' />}
+      {(!model._isPending() && !isPinned) && model.number}
     </div>
-  )
-})
+    <div className='command-expander-column' onClick={() => model.toggleOpen()}>
```

Comment:
It seems like this command-expander-column element is eating clicks when the command isn't actually toggle-able/has no children. We could selectively add the onClick based on child presence as well?